charts/osm/values.yaml

```diff
@@ -43,6 +43,7 @@ OpenServiceMesh:
   meshName: osm
   useHTTPSIngress: false
   envoyLogLevel: error
+  osmLogLevel: trace
```

Comment:
should we rename this to `controllerLogLevel`?

Reply:
Agreed, otherwise it may be confused with envoy log level

Reply:
... I guess? we didn't call `envoyLogLevel` `sidecarProxyLogLevel`, but if you feel this is clearer, I'll change it gladly

Reply:
since we use it as `OpenServiceMesh.controllerLogLevel`, seems more than ok. changing it.